test/e2e/run-e2e-test.js

```diff
@@ -63,8 +63,17 @@ async function main() {
     throw error;
   }
 
+  let command = 'yarn';
+  const args = ['mocha', '--no-timeouts', e2eTestPath];
+
+  // Run test with virtual display on CI
+  if (process.env.CI === 'true') {
+    command = 'xvfb-run';
```

Comment:
Seemingly the Firefox E2E tests don't like this :thinking: 

I'm not sure why, given that Xvid is already running at this point. And this shouldn't conflict - the `-a` flag ensures it finds a free port to use. We can consider using this just for Chrome, but.... it's unsettling not understanding what's going on here.

In the GitHub Actions migration PR I placed this in the CI conflig directly, rather than here. It was needed for Firefox there, since the GItHub Actions base image doesn't run `Xvfb` automatically like the CircleCI one does.

I have no idea what impact running it per-test-suite has v.s. once for all test suites. If it's not shutting down properly or starting up fast enough, that could explain some crashes.